browser/ui/webui/brave_settings_ui.cc

```diff
@@ -101,6 +102,9 @@ void BraveSettingsUI::AddResources(content::WebUIDataSource* html_source,
   html_source->AddBoolean(
       "isDeAmpFeatureEnabled",
       base::FeatureList::IsEnabled(de_amp::features::kBraveDeAMP));
+  html_source->AddBoolean(
+      "isBraveRewardsSupported",
```

Comment:
In the extension code this is called `isSupported`. I think we should use consistent naming and also not duplicated the logic. Maybe add RewardsService::IsSupported and use that?

Reply:
but actually why are we showing this UI in the first place if rewards is disabled by policy?

Reply:
This is for Settings UI, isSupported will be unclear as this the the entire settings UI, this bool is needed to hide Rewards related items.